book/src/5.in-depth-guides/4.auth/4.signin.md

```diff
@@ -0,0 +1,9 @@
+# Initiating Signing In
+
+Prompting the user to sign in using an `OAuth` connection has
```

Comment:
Add the fact that the oauth cards can be customized

Reply:
Consider showing this as well.

Reply:
will include in ff